examples/SampleApp/src/utils/getUserActivityStatus.ts

```diff
@@ -14,7 +14,7 @@ export const getUserActivityStatus = (
   if (!user) return '';
 
   if (user.online) {
-    return `Online for ${Dayjs(user?.last_active).toNow(true)}`;
+    return `Online`;
```

Comment:
is your linter off? this should've said it needs to be `'` instead

Reply:
hmm good catch!! Strange that pre-commit hook didn't catch it